src/LtiMessageLaunch.php

```diff
@@ -88,6 +89,10 @@ public function __construct(
         $this->cache = $cache;
         $this->cookie = $cookie;
         $this->serviceConnector = $serviceConnector;
+
+        if ($this->serviceConnector && $serviceConnectorDebuggingMode) {
+            $this->serviceConnector->setDebuggingMode($serviceConnectorDebuggingMode);
```

Comment:
I'm not a fan of how this looks. Why are we doing it this way instead of just relying on the code that passes in the service connector to set the debugging mode?

Reply:
@dbhynds I tried setting the debugging mode of the service connector in the provider, but that seemed to be separate from the `serviceConnector` referenced internally by the `LtiMessageLaunch`. However, I've done some refactoring to instead expose a function which toggles the debugging mode directly; implementation on the app side is done in [this questions PR](https://github.com/packbackbooks/questions/pull/8199/files), what do you think of this approach?